src/cf-tables.less

```diff
@@ -370,11 +380,24 @@
 }
 
 .table__sortable {
+    button.sortable {
+        width: 100%;
+        height: 100%;
+        padding: inherit;
+        margin: inherit;
+        border: none;
+        background: none;
+        font-family: inherit;
+        font-weight: inherit;
+        outline: none;
+        text-align: left;
+    }
```

Comment:
Could this be simplified by mixing in the [`.btn__link` modifier](http://cfpb.github.io/cf-buttons/docs/#button-link from cf-buttons?